.gitpod.yml

```diff
@@ -0,0 +1,7 @@
+tasks:
+  - name: Initialize Poetry Environment
```

Comment:
This will become the name of the terminal itself, is it suitable for that though?

Reply:
I think so

Reply:
I'd suggest something more like just a bare:
```suggestion
  - name: Python Environment
```

Reply:
I'll do this, but to clarify this only sets this as the terminal name while the commands are running